src/components/tutorial-meta/components/variant-list/index.tsx

```diff
@@ -17,10 +17,13 @@ export function VariantList() {
 
 	return (
 		<>
-			<div className={s.desktopVariantList}>
+			<div className={s.desktopVariantList} data-heap-track="variant-list">
 				<DesktopVariantList variant={currentVariant} />
 			</div>
-			<div className={s.mobileVariantDropdownDisclosure}>
+			<div
+				className={s.mobileVariantDropdownDisclosure}
+				data-heap-track="variant-list"
+			>
```

Comment:
Will validate this in heap after its merged. Only one of these shows at a time depending on viewport width.

Reply:
💭 actually im going to remove one, we don't need both